scripts/time_plots.py

```diff
@@ -0,0 +1,81 @@
+import json
```

Comment:
Put `#!/usr/bin/env python3` on the first line of the file, then a blank line. Also, mark time_plots.py as executable by running `git update-index --chmod=+x time_plots.py`.